### ISSUE TITLE
Facebook standards events not being tracked

### ISSUE THREAD

**Issue by RicardoRB** [oddbit/flutter_facebook_app_events#86]
All the events are showing in my dashboard but not the standards events as "purchase" or "checkout"

## Expected Behavior
Tracking Facebook standards events

## Current Behavior
Not tracking standards events

## Steps to Reproduce (for bugs)
1. Track any standard event
2. Check your dashboard

## Context
I can not tracking what user add to cart or purchase

## Your Environment
[√] Flutter (Channel stable, 1.22.6, on Microsoft Windows [Version 10.0.19042.804], locale es-ES)
    • Flutter version 1.22.6 at C:\tools\flutter
    • Framework revision 9b2d32b605 (6 weeks ago), 2021-01-22 14:36:39 -0800
    • Engine revision 2f0af37152
    • Dart version 2.10.5

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\ricar\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    X Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/windows#android-setup for more details.

[!] Android Studio (version 4.1.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    X Flutter plugin not installed; this adds Flutter specific functionality.
    X Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] VS Code (version 1.54.1)
    • VS Code at C:\Users\ricar\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.20.0

[√] Connected device (1 available)
    • Lenovo P2a42 (mobile) • ZY223SXWKH • android-arm64 • Android 7.0 (API 24)

! Doctor found issues in 2 categories.

**Comment by naoaki-kaito** [comment]
I had the same problem.
I'm using `logEvent` alternatively for now.

example: 
```
FacebookAppEvents().logEvent(
  name: "fb_mobile_content_view",
      parameters: {
        "content": 'XXX',
        "currency": 'XXX',
        "value": 'XXX',
  },
);
```

app events names are following.
[Collaborative Ads App Events API](https://developers.facebook.com/docs/marketing-api/app-event-api/app-events-api-for-collaborative-ads/)

**Comment by RicardoRB** [comment]
> I had the same problem.
> 
> I'm using `logEvent` alternatively for now.
> 
> 
> 
> example: 
> 
> ```
> 
> FacebookAppEvents().logEvent(
> 
>   name: "fb_mobile_content_view",
> 
>       parameters: {
> 
>         "content": 'XXX',
> 
>         "currency": 'XXX',
> 
>         "value": 'XXX',
> 
>   },
> 
> );
> 
> ```
> 
> 
> 
> app events names are following.
> 
> [Collaborative Ads App Events API](https://developers.facebook.com/docs/marketing-api/app-event-api/app-events-api-for-collaborative-ads/)
> 
> 
> 
> 
> 
> 

Thanks! I'll try it as a temporary solution

**Comment by DennisAlund** [comment]
Thank you for the report. 

Not sure I fully understand what the scope of the problem is? You are missing some event methods?

Regarding the one you mentioned in the example, am I understanding it correctly that you are having an issue with the `logViewContent()` method as described in #73?

**Comment by DennisAlund** [comment]
If the issue is that you're missing some alias methods for logging events, there are a a few paths to take on that
 1. Make a list of the exact methods you'd want/need and wait for me or someone to implement it.
 2. Make a pull request with said methods implemented 🤗 
 3. In the meanwhile, you can create extension methods to implement them in your own project, while waiting. See comments made in this PR: https://github.com/oddbit/flutter_facebook_app_events/pull/67#issuecomment-784637262

**Comment by RicardoRB** [comment]
> Thank you for the report.
> 
> Not sure I fully understand what the scope of the problem is? You are missing some event methods?
> 
> Regarding the one you mentioned in the example, am I understanding it correctly that you are having an issue with the `logViewContent()` method as described in #73?

The problem is that I can't see the standard events in my Facebook dashboard (fb_mobile_purchase, fb_mobile_add_to_cart, fb_mobile_initiated_checkout). Those are not possible to be tracked with Facebook campaigns because are not getting triggered. The rest are showing in my Facebook Analytics.

I'm not sure what kind of problem can it be, I'll try with as @naoaki-kaito suggested, and I'll come back to report it as a temporary solution.

**Comment by RicardoRB** [comment]
> I had the same problem.
> I'm using `logEvent` alternatively for now.
> 
> example:
> 
> ```
> FacebookAppEvents().logEvent(
>   name: "fb_mobile_content_view",
>       parameters: {
>         "content": 'XXX',
>         "currency": 'XXX',
>         "value": 'XXX',
>   },
> );
> ```
> 
> app events names are following.
> [Collaborative Ads App Events API](https://developers.facebook.com/docs/marketing-api/app-event-api/app-events-api-for-collaborative-ads/)

I tried but still doesn't work, have you done something special? Do you use "await" to track it? 

An example of what I have:

```dart
_facebookAppEvents.logEvent(
      name: "fb_mobile_add_to_cart",
      parameters: {
        "fb_content": [
          {"id": itemId, "quantity": quantity},
        ],
        "content_type": itemCategory,
        "fb_currency": currency,
      },
      valueToSum: price,
    );
```

**Comment by naoaki-kaito** [comment]
@RicardoRB 

`fb_content` must be String.

```
"fb_content":  "[{\"id\": \"$itemId\", \"quantity\": $quantity}]"
```

**Comment by DennisAlund** [comment]
https://github.com/oddbit/flutter_facebook_app_events/blob/d8e423be323bd7f5e6163aa75b70bd82a03ecb38/lib/facebook_app_events.dart#L33-L37

**Comment by DennisAlund** [comment]
@RicardoRB please update to `v0.11.1` where add to cart as been added

https://github.com/oddbit/flutter_facebook_app_events/blob/8d27fb486de2b597d9190d170bf877b943796bdc/lib/facebook_app_events.dart#L214-L231

**Comment by DennisAlund** [comment]
Solved in version: 0.11.2

**Comment by RicardoRB** [comment]
Thanks @DennisAlund and @naoaki-kaito now it works pretty fine. I had to use the exactly variables that the events also need 😃
